continuous_queryies.md

```diff
@@ -0,0 +1,104 @@
+# Continuous Queries
+
+This document lays out continuous queries and a proposed architecture for how they'll work within an InfluxDB cluster.
+
+## Definition of Continuous Queries
+
+Continuous queries serve two purposes in InfluxDB: denormalizing data into many series or into a single series and aggregating and downsampling data. We refer to these as `fan-out`, `fan-in`, or `aggregation`. Before we get to the continuous query examples, we need to define the `INTO` syntax of queries.
+
+### INTO
+
+`INTO` is a method for running a query and having it output into either another measurement name, retention policy, or database. The syntax looks like this:
+
+```sql
+SELECT *
+INTO [<retention policy>.]<measurement> [ON <database>]
+FROM <measurement>
+[WHERE ...]
+[GROUP BY ...]
+```
+
+The syntax states that the retention policy, database, where clause, and group by clause are all optional. If a retention policy isn't specified, the database's default retention policy will be written into. If the database isn't specified, the database the query is running from will be written into.
+
+By selecting specific fields, `INTO` can merge many series into one that will go into a new either a new measurement, retention policy, or database. For example:
+
+```sql
+SELECT mean(value) as value, region
+INTO 1h.cpu_load
+FROM cpu_load
+GROUP BY time(1h), region
+```
+
+That will give 1h summaries of the mean value of the `cpu_load` for each `region`. Specifying `region` in the `GROUP BY` clause is unnecessary since having it in the `SELECT` clause forces it to be grouped by that tag, we've just included it in the example for clarity.
+
+With `SELECT ... INTO`, fields will be written as fields and tags will be written as tags.
+
+### Continuous Query Syntax
+
+The `INTO` queries run once. Continuous queries will turn `INTO` queries into something that run in the background in the cluster. They're kind of like triggers in SQL.
+
+```sql
+CREATE CONTINUOUS QUERY 1h_cpu_load
+ON database_name
+BEGIN
+  SELECT mean(value) as value, region
+  INTO 1h.cpu_load
+  FROM cpu_load
+  GROUP BY time(1h), region
+END
+```
+
+Or chain them together:
+
+```sql
+CREATE CONTINUOUS QUERY 10m_event_count
+ON database_name
+BEGIN
+  SELECT count(value)
+  INTO 10m.events
+  GROUP BY time(10m)
+END
+
+-- this selects from the output of one continuous query and outputs to another series
+CREATE CONTINUOUS QUERY 1h_event_count
+ON database_name
+BEGIN
+  SELECT sum(count) as count
+  INTO 1h.events
```

Comment:
Yeah, there should still be a `FROM` otherwise you don't know what measurement it's coming from.